tests/cypress/support/commands.ts

```diff
@@ -783,42 +783,74 @@ Cypress.Commands.add('checkGitRepoAfterUpgrade', (repoName, fleetNamespace='flee
   cy.verifyTableRow(0, /Active|Modified/, repoName);
 });
 
+Cypress.Commands.add('currentClusterResourceCount', (clusterName) => {
+  cy.accesMenuSelection('Continuous Delivery', 'Clusters');
+  cy.contains('.title', 'Clusters').should('be.visible');
+  cy.filterInSearchBox(clusterName);
+  cy.verifyTableRow(0, 'Active', clusterName);
+  cy.get('td.col-link-detail > span').contains(clusterName).click();
+  cy.get("div[primary-color-var='--sizzle-success'] div[class='data compact'] > h1")
+  .invoke('text')
+  .then((clusterResourceCountText) => {
+    // Convert to integer
+    const clusterCurrentResourceCount = parseInt(clusterResourceCountText.trim(), 10);
+    cy.log("Resource count on each cluster is: " + clusterCurrentResourceCount);
+    cy.wrap(clusterCurrentResourceCount).as('clusterCurrentResourceCount');
+  })
+})
+
 Cypress.Commands.add('gitRepoResourceCountAsInteger', (repoName, fleetNamespace='fleet-local') => {
   cy.accesMenuSelection('Continuous Delivery', 'Git Repos');
   cy.fleetNamespaceToggle(fleetNamespace);
   cy.verifyTableRow(0, 'Active', repoName);
   cy.contains(repoName).click()
   cy.get('.primaryheader > h1').contains(repoName).should('be.visible')
 
-  // Get the Resource count text from UI and convert it into integer.
   cy.get("div[data-testid='gitrepo-deployment-summary'] div[class='count']")
   .invoke('text')
-  .then((countText) => {
-    // Add '7' default resource count available on each cluster.
-    const gitRepoResourceCount = parseInt(countText.trim(), 10) + 7;
```

Comment:
Where did this `+7` went?

Reply:
It will be dynamically calculated with this command: https://github.com/rancher/fleet-e2e/pull/288/files#diff-75965941f29be613c2bab5bdb432b711b86c52a2567d7f08dcf8aaeef907eb0cR786-R801

Reply:
There is default resources difference on 2.10 and 2.11 onwards.
By default 7 resources are present on cluster 2.10 but later versions it is showing 6 resources.